cms/templates/course_outline.html

```diff
@@ -175,6 +175,31 @@ <h2 class=status-pacing-label>${_("Course Pacing")}</h2>
                     % endif
                 </div>
                 <div class="status-highlights-enabled"></div>
+                % if should_show_checklists_page(request.user):
+                <div class="status-checklist">
+                    <%static:studiofrontend entry="courseOutlineHealthCheck">
+                        {
+                            "lang": "${context_course.language | n, js_escaped_string}",
```

Comment:
I wonder why `lang` is determined differently between the two templates.

Here, it's `context_course.language`, but in `checklists.html` it's `language_code`.

I wonder if we can pass the language_code in [here](https://github.com/edx/edx-platform/blob/bea86a673a67c6f363ce9e1f2b3ec2be831a0969/cms/djangoapps/contentstore/views/course.py#L639) the same way we do with `checklists.html`?